deb/ubuntu-focal/Dockerfile

```diff
@@ -0,0 +1,41 @@
+ARG GO_IMAGE
+ARG DISTRO=ubuntu
+ARG SUITE=focal
+ARG BUILD_IMAGE=${DISTRO}:${SUITE}
+
+FROM ${GO_IMAGE} AS golang
+
+FROM ${BUILD_IMAGE}
+
+# Remove diverted man binary to prevent man-pages being replaced with "minimized" message. See docker/for-linux#639
+RUN if  [ "$(dpkg-divert --truename /usr/bin/man)" = "/usr/bin/man.REAL" ]; then \
+        rm -f /usr/bin/man; \
+        dpkg-divert --quiet --remove --rename /usr/bin/man; \
+    fi
+
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
not in eoan's dockerfile

Reply:
otherwise lgtm

Reply:
Correct; it's needed with Ubuntu 20.04 and up

They changed the contents of the `devscripts` package to include `tzutils`, which doesn't install otherwise (waiting for input to set your timezone); https://github.com/docker/containerd-packaging/pull/142#issuecomment-603128770

And https://twitter.com/thaJeztah/status/1242413653913108480?s=20

<img width="608" alt="Screenshot 2020-04-11 at 02 08 40" src="https://user-images.githubusercontent.com/1804568/79030470-6997f300-7b99-11ea-9131-55d8468fecdc.png">

Reply:
I also experienced this and can confirm. As described in [that comment](https://github.com/docker/containerd-packaging/pull/142#issuecomment-603128770) linked above,  the build hangs in the `apt-get install` step. `tzdata` wants you to select a time zone with the keyboard, but there is no way to actually select one, even manually. (And certainly CI can't rely on human input, even if you could do it manually.)

Presumably, the `DEBIAN_FRONTEND=noninteractive` lets the package configure this setting itself, using a default timezone value perhaps. In any case, it successfully works around the hang.